### ISSUE TITLE
Test results?

### ISSUE THREAD

**Issue by kxhit** [qiqihaer/votenet-kitti#1]
Thanks for your code! Have you evaluated the test results or evaluation results?

**Comment by qiqihaer** [comment]
No, I have not evaluated on the test set. The parameters for training has not been tuned.

**Comment by turboxin** [comment]
Hi @qiqihaer, could please share the results on validation set? Thanks a lot!
